docker-compose.yml

```diff
@@ -119,13 +119,13 @@ services:
         +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
         3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
         -----END RSA PRIVATE KEY-----
-      - LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS=000000004400001=lab_results_mailbox
-          000000024600002=lab_results_mailbox
-          000000183500005=lab_results_mailbox
-          110000044300001=lab_results_mailbox
-      - LAB_RESULTS_MESH_POLLING_CYCLE_MINIMUM_INTERVAL_IN_SECONDS=300
-      - LAB_RESULTS_MESH_POLLING_CYCLE_DURATION_IN_SECONDS=285
-      - LAB_RESULTS_MESH_CLIENT_WAKEUP_INTERVAL_IN_MILLISECONDS=60000
+      - LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS=000000004400001=lab_results_ack_mailbox
```

Comment:
Why this change? Shouldn't this stay as `lab_results_mailbox`?

Reply:
That's for testing purpose only. Having the same mailbox for both interchanges and nhsacks adds a lot of noise when testing.